ambari-logsearch/docker/bin/start.sh

```diff
@@ -56,6 +56,12 @@ function create_logsearch_configs() {
   else
     cp /root/test-config/logsearch/logsearch.properties /root/config/logsearch/logsearch.properties
   fi
+
+  if [ $KNOX == 'true'  ]
```

Comment:
```
/root/start.sh: line 60: [: ==: unary operator expected
```

This happens if `Profile` is already created before this change (ie. `KNOX` is undefined).  Same for a few other usage `$KNOX` below.

I guess it would be nice to set default values at the start of the script to cover "upgrade" scenario.  OK to do in another PR, since it probably affects other profile variables, too.

Reply:
or i could just use quotes, that should solve that problem